payments/swish/other-features.md

```diff
@@ -157,6 +157,21 @@ Content-Type: application/json
 }
 ```
 
+### Mobile Number Validation
+
+#### eCommerce
+
+All international mobile numbers are supported. To be valid, the number input
+must be with a country code prefix and consist of 8 to 15 characters. Digits are
+the only characters allowed, and the regex used is `\\+[1-9]\\d{7,14}`. A valid
+Swedish mobile number would be +46700123123, a valid Norwegian mobile number
```

Comment:
Do we perhaps have a test data phone number we can use here? And if we do, should we perhaps define it as a new global variable that is used both places?

```suggestion
Swedish mobile number would be `+46700123123`, a valid Norwegian mobile number
```